config/paths-osx.conf

```diff
@@ -25,3 +25,5 @@ syslog_authpriv = /var/log/secure.log
 
 #syslog_local0 =
 
+# Default Mac OS log location for syslog output.
```

Comment:
this changes are not required if you used direct specified `logpath` in `jail.conf`